reactioncommerce/reaction
# feat: navigation manager UI

Resolves #4935  
Impact: **minor**  
Type: **feature**

Original work from @ajporlante

## Issue

An interface is needed to allow admins to modify the navigation

## Solution

Build a navigation editor UI in the Operator 2.0 admin panel

## Breaking changes

none

## Testing
1. Add, remove, update navigation items
2. See changes applied to the storefront using this PR: https://github.com/reactioncommerce/reaction-next-starterkit/pull/472

Note. Saving navigation items, and the tree publishes immediately. Discard only discards unsaved changes.
